### PR TITLE
integrated backend for profile editing

### PR DESCRIPTION
**Pull request by jeromes123**
**OVERVIEW**
Removed unnecessary code for editpost.js
Modified controller codes to suit view

### CODE REVIEW

File: src/resources/views/components/modals/edit-profile.blade.php [hunk 1]
```diff
@@ -1,14 +1,16 @@
 <div onclick="hideProfile()" id="profile"
     class="fixed left-0 top-0 bg-mydark bg-opacity-50 w-full h-full justify-center items-center opacity-0 hidden transition-opacity duration-500">
     <div onclick="event.stopImmediatePropagation()" class="bg-mycream rounded-2xl shadow-md p-10 flex">
+        <form method="POST" action="/update-profile">
```
Comment (mikco-hipe) [inline src/resources/views/components/modals/edit-profile.blade.php]: hmm update profile but the method is POST?

**Review by mikco-hipe — COMMENTED** [review]
Please check!

File: src/resources/views/components/modals/edit-profile.blade.php (hunk 1 above)
Comment (hipe01mark) [reply src/resources/views/components/modals/edit-profile.blade.php]: +1

Comment (hipe01mark) [reply src/resources/views/components/modals/edit-profile.blade.php]: also use route() with name when calling end points

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Pull request merged**